modules/openapi-generator/src/main/resources/python-legacy/asyncio/rest.mustache

```diff
@@ -62,7 +62,8 @@ class RESTClientObject(object):
 
         # https pool manager
         self.pool_manager = aiohttp.ClientSession(
-            connector=connector
+            connector=connector,
+            trust_env=True
```

Comment:
This option causes that the client gets proxies information from HTTP_PROXY / HTTPS_PROXY environment variables. It works in this way for legacy and new python client (requests library) and it's expected behavior.